src/containers/Root/AuthorizedRoot.tsx

```diff
@@ -12,7 +14,10 @@ const Wrapper = styled.div`
 export const AuthorizedRoot: React.FC = () => {
   return (
     <Wrapper>
-      <Header onLogoutClick={ () => {}} onChangeLanguage={ () => {} } userNameInitials='RT'/>
+      <Header onLogoutClick={ () => {} } onChangeLanguage={ () => {} } userNameInitials='RT'/>
```

Comment:
looks, that you should pass full name here. because you use first letters from the string.